src/developer/web-api/tracker.md

```diff
@@ -2924,6 +2924,12 @@ Tracker objects, the requested organisation unit must always be within the user'
 user cannot request objects outside its search scope unless it's using the organisation unit mode
 ALL and has enough privileges to use that mode.
 
+When requesting tracked entities without specifying a program, the response will include only those tracked entities that satisfy one of the following criteria:
+- The tracked entity is enrolled in at least one program the user has data access to, and the user has access to the owner organisation unit.
+- The tracked entity is not enrolled in at least one program the user has data access to, and the user has access to the tracked entity registering organisation unit.
```

Comment:
That sounds odd 🤔 

`The tracked entity is not enrolled in at least one program the user has data access to`

Can you phrase this differently? I don't I get it 😅

Reply:
it should be a bit more clear now